app/services/search_service.rb

```diff
@@ -318,25 +322,23 @@ def search_kick_where_from_filters(filters: [], skip_filters: [], is_crowd_agg:
   def find_visibile_options(agg_name, is_crowd_agg, current_site, url, config_type, return_all)
     return [] if current_site.blank? || return_all
 
-    if !url || url.empty?
-      view = current_site.site_views.find_by(default: true).view
-    else
+    view = if url.blank?
```

Comment:
most of this is related to autoformatting. ladies and gents, please autoformat your code according to the rubocop specifications already set in the project.